.github/workflows/compiler-zoo.yml

```diff
@@ -38,11 +30,12 @@ jobs:
       run: |
         sudo apt-get update
         sudo apt-get -yq --force-yes install ${{ matrix.zoo.cc }}
+        sudo apt-get -yq --force-yes install gcc-multilib
```

Comment:
Uhm, how are the different clang versions installed?

Reply:
By the command above?

Reply:
`apt-get install clang-12` and similar.  They are later invoked including the version so it should all be happy.